device_plugins/deploy_qat.md

```diff
@@ -26,4 +38,12 @@ Output:
 ```
 NAME		        DESIRED		READY	NODE SELECTOR	                                    AGE
 qatdeviceplugin-sample  1 	        1       {"intel.feature.node.kubernetes.io/qat":"true"}     3h27m
+```
+
+## verify QAT resource is available
+QAT has two kinds of resource `qat.intel.com/cy` for cryptography and `qat.intel.com/dc` for compression. If the plugin deployment is a success, the resources can be seen using the commands below. Use /dc is compression is configured.
```

Comment:
Can this section be written in a way that `cy`and `dc` are not explicitly mentioned. We *might* have to change these in the future when the 6.5 features become available. Maybe just say "look for `qat.intel.com` resources for compression and crypto"

Reply:
modified